Fix for dependency error

prashanthmadi opened the pull request:
::     FAILED DOWNLOADS            ::
                ::::::::::::::::::::::::::::::::::::::::::::::
                :: com.hortonworks#shc-core;1.1.0-2.1-s_2.11!shc-core.jar
                ::::::::::::::::::::::::::::::::::::::::::::::

azurecla commented:
Hi __@sauryadas__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (saudas). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

sauryadas commented:
need to make this request to the azure-content-pr

prashanthmadi committed:
Fix for dependency error

                ::     FAILED DOWNLOADS            ::
                ::::::::::::::::::::::::::::::::::::::::::::::
                :: com.hortonworks#shc-core;1.1.0-2.1-s_2.11!shc-core.jar
                ::::::::::::::::::::::::::::::::::::::::::::::
